src/layers/viv-viewer-layer/viv-viewer-layer-base.js

```diff
@@ -0,0 +1,62 @@
+import { TileLayer } from '@deck.gl/geo-layers';
+import { COORDINATE_SYSTEM } from '@deck.gl/core';
+
+import { loadZarr, loadTiff } from './data-utils';
+import { inTileBounds, renderSubLayers } from './utils';
+
+const defaultProps = {
+  ...TileLayer.defaultProps,
+  pickable: false,
+  coordinateSystem: COORDINATE_SYSTEM.CARTESIAN,
+  sliderValues: { type: 'array', value: [], compare: true },
+  colorValues: { type: 'array', value: [], compare: true },
+  tileSize: { type: 'number', value: 512, compare: true },
+  imageHeight: { type: 'number', value: 1028, compare: true },
+  imageWidth: { type: 'number', value: 1028, compare: true },
```

Comment:
```suggestion
  imageHeight: { type: 'number', value: 1024, compare: true },
  imageWidth: { type: 'number', value: 1024, compare: true },
```
... which is an argument for using exponents. :)